packages/@azure/servicebus/data-plane/test/testUtils.ts

```diff
@@ -383,3 +384,18 @@ export async function purge(
     }
   }
 }
+
+// Maximum wait duration for the Streaming Receiver to receive the messages = `10000 ms`(10 seconds)(= maxWaitTime)
+// Keep checking whether the predicate is true after every `1000 ms`(1 second) (= delayBetweenRetries)
+export async function DelayStreaming(
```

Comment:
Let's follow camel casing and name it as `delayStreaming`. We follow this for all other function names

Reply:
You might want to make the name more generic and remove `Streaming`, since the function itself is a generic timer and predicate operation.  We may also want to (eventually) share this helper function across all our tests in the repo, in which case the generic name is even more important.

Reply:
Good point. How about `waitUntil`?

Reply:
Yes, thought the same.
How about CheckWithTimeout ?

Reply:
`checkWithTimeout`